design-system/react/src/components/InputAmount/InputAmount.test.tsx

```diff
@@ -0,0 +1,80 @@
+import type { BN } from '@fuel-ts/math';
+import { bn } from '@fuel-ts/math';
+import { fireEvent, render, screen, testA11y } from '@fuel-ui/test-utils';
+import { useState } from 'react';
+
+import type { InputAmountProps } from './InputAmount';
+import { InputAmount } from './InputAmount';
+
+export const MOCK_ASSET = {
+  assetId: '0x0000000000000000000000000000000000000000000000000000000000000000',
+  amount: bn(14563943834),
+};
+const FIELD_NAME = 'Input Amount';
+
+function Content({ onChange, ...props }: Partial<InputAmountProps>) {
+  const [val, setVal] = useState<BN>(bn(0));
+
+  function handleChange(val: BN) {
+    onChange?.(val);
+    setVal(val);
+  }
+
+  return (
+    <InputAmount
+      name={FIELD_NAME}
+      balance={MOCK_ASSET.amount}
+      onChange={handleChange}
+      value={bn(val)}
+      {...props}
+    />
+  );
+}
+
+describe('InputAmount', () => {
+  const onChange = jest.fn();
+
+  it('a11y', async () => {
+    await testA11y(<Content onChange={onChange} />);
+  });
```

Comment:
can we also `a11y` the Loader component ?

Reply:
We already have